puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/debian/stage_packages.sh

```diff
@@ -11,7 +11,7 @@ DEB_PATH=./debian/${os}/build-${project}
 
 # Upload all builds to stagingdeb for testing
 echo "scratch build: uploading to stagingdeb/${os}/${repoowner}-${version}"
-export RSYNC_RSH="ssh -i /var/lib/workspace/workspace/staging_key/rsync_freightstage_key"
+export RSYNC_RSH="ssh -i /home/jenkins/workspace/staging_key/rsync_freightstage_key"
```

Comment:
Ideally we'd use Jenkins secrets similar to 066cfae0ea07d5c15fc2488853971c8f609f21e8 and ad703c92d5dd12599c044ff438c4d0aefe9b24bb for this.